asmcomp/asmlink.ml

```diff
@@ -373,61 +373,68 @@ let link ~ppf_dump objfiles output_name =
 (* Error report *)
 
 open Format
+module Style = Misc.Color
```

Comment:
Should we move the parts of the Color module that are really about Style to a new submodule Misc.Style, instead of hinting at the better API in user code only?

Reply:
I would rather propose to move the whole `Misc.Color` submodule to a new `Style` module outside of `Misc`.

Reply:
After rereading the PR, I did end up splitting `Misc.Color` into `Misc.Color` and `Misc.Style`.